runtime/wasmer_test.go

```diff
@@ -724,3 +748,101 @@ func TestExt_free(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+// utility function to create a test block
+func createTestBlock() (*common.Block, error) {
+	parentHash, err := common.HexToHash("0x4545454545454545454545454545454545454545454545454545454545454545")
+	if err != nil {
+		return nil, err
+	}
+
+	stateRoot, err := common.HexToHash("0xb3266de137d20a5d0ff3a6401eb57127525fd9b2693701f0bf5a8a853fa3ebe0")
+	if err != nil {
+		return nil, err
+	}
+
+	extrinsicsRoot, err := common.HexToHash("0x03170a2e7597b7b7e3d84c05391d139a62b157e78786d8c082f29dcf4c111314")
+	if err != nil {
+		return nil, err
+	}
+
+	testHeader := common.BlockHeader{
+		ParentHash:     parentHash,
+		Number:         big.NewInt(1),
+		StateRoot:      stateRoot,
+		ExtrinsicsRoot: extrinsicsRoot,
+		Digest:         []byte{},
+	}
+
+	testBlock := common.Block{
+		Header:     &testHeader,
+		Extrinsics: []byte{},
+	}
+	return &testBlock, err
+}
+
+func TestCallCoreExecuteBlock(t *testing.T) {
+	r, err := newRuntime(t)
+	if err != nil {
+		t.Fatal(err)
+	} else if r == nil {
+		t.Fatal("did not create new VM")
+	}
+
+	mem := r.vm.Memory.Data()
+
+	// sample test data from substrait test
+	data := []byte{69, 69, 69, 69, 69, 69, 69, 69, 69, 69, 69, 69, 69, 69, 69, 69, 69, 69, 69, 69, 69, 69, 69, 69, 69, 69, 69, 69, 69, 69, 69, 69, 4, 179, 38, 109, 225, 55, 210, 10, 93, 15, 243, 166, 64, 30, 181, 113, 39, 82, 95, 217, 178, 105, 55, 1, 240, 191, 90, 138, 133, 63, 163, 235, 224, 3, 23, 10, 46, 117, 151, 183, 183, 227, 216, 76, 5, 57, 29, 19, 154, 98, 177, 87, 231, 135, 134, 216, 192, 130, 242, 157, 207, 76, 17, 19, 20, 0, 0}
```

Comment:
can you link to the substrate test code?

Reply:
I added a `println!("b.encode: {:?}", &b.encode());`  before this line to get the sample data.

https://github.com/paritytech/substrate/blob/203e2c23b51bf23f778b3afee16465e6dca04cf2/core/test-runtime/src/system.rs#L373

Reply:
okay cool